ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -698,6 +1073,46 @@ def _collectOptPoint(self, rlz, fitness, objectiveVal, g):
 
     return point
 
+  def _collectOptPointMulti(self, population, rank, CD, objVal, fitness, constraintsV):
+    """
+      Collects the point (dict) from a realization
+      @ In, population, Dataset, container containing the population
+      @ In, objectiveVal, list, objective values at each chromosome of the realization
+      @ In, rank, xr.DataArray, rank values at each chromosome of the realization
+      @ In, crowdingDistance, xr.DataArray, crowdingDistance values at each chromosome of the realization
+      @ Out, point, dict, point used in this realization
```

Comment:
Docstring need to be updated here

Reply:
Fixed f37b0968ea845b6559d455cfd1e95760b665b683